qiita_pet/static/vendor/js/resumable-uploader.js

```diff
@@ -117,11 +117,8 @@
      // Add a new file (or rather: glue between newly added resumable files and the UI)
      this.addFile = function(resumableFile){
        // A list and and edit item for the UI
-       if (typeof resumableFile == 'object') {
-         name = resumableFile.fileName
-       } else {
-         name = resumableFile
-       }
+       fileName = resumableFile.fileName
```

Comment:
In the past we were passing str (already uploaded) or object (just uploaded) and now we are always passing an object.